.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: python
+
+sudo: false
+
+python:
+  - "3.5"
+  - "3.6"
+
+install:
+  - pip install -U -r requirements.txt
```

Comment:
`-U` is not what we want. Because it will break a lot of versions. We try to fix them with glue.